src/components/dev-hub/survey-banner.js

```diff
@@ -0,0 +1,113 @@
+import React, { useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import Button from './button';
+import Icon from '@leafygreen-ui/icon';
+import { fontSize, screenSize, size, colorMap } from './theme';
+
+const Footer = styled('div')`
```

Comment:
One last note, can we give this a `z-index` of `superFront` from the theme? When QA testing I noticed it would go behind some content like this:

![Screen Shot 2021-07-07 at 5 20 20 PM](https://user-images.githubusercontent.com/9064401/124830271-a17e1980-df47-11eb-87a1-e1f51498247c.png)

Reply:
Okay, yeah haven’t pushed my changes yet.

Reply:
👍